scripts/build/beta.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -ev
+
+GOPATH=$(go env GOPATH)
+REPO_DIR=${GOPATH}/src/github.com/algorand/go-algorand
+cd ${REPO_DIR}
+
+# Run `make` to ensure `buildtools` is available
+make -j4
+
+# Flag that we want release handling of genesis files
+export RELEASE_GENESIS_PROCESS=true
+
+git checkout rel/beta
+
+# Disabled because we have static genesis files now
```

Comment:
for this commented out block, cleanup rather than replicate?

Reply:
Sorry - just saw this.  If i have to update again I will clean it up